eng/Signing.props

```diff
@@ -9,6 +9,8 @@
     <FileSignInfo Include="Nerdbank.Streams.dll" CertificateName="3PartySHA2" />
     <FileSignInfo Include="MessagePack.Annotations.dll" CertificateName="3PartySHA2" />
     <FileSignInfo Include="MessagePack.dll" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="Humanizer.dll" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="ICSharpCode.Decompiler.dll" CertificateName="3PartySHA2" />
```

Comment:
This seems scary in that it implies we are packaging roslyn DLLs (and references). Do we know why this needs to be added?

Reply:
I'm not sure tbh. Let me look through the binlog and find out

Reply:
If these dlls don't end up in the nupkg(s) then I imagine we're good, but definitely would be curious to know the story.

Reply:
Agreed! Great point @davidwengier

Reply:
Alright, I dug into this. We've always shipped this binary with rzls. It's in the current extension. I don't know why there's a difference in signing info now, but this seems correct? We pull it in when we pull in `Microsoft.CodeAnalysis.CSharp.Workspaces`.

I'm investigating still why this is flagged. It may be the changes I made to have the package be delivered from a different folder, or some of the changes to `IsShipping`

Reply:
If we always shipped it, then I'm fine with this. I guess we never did that work to de-dupe files in the extension as a whole.

Reply:
With co-hosting, rzls can go away. So, so this seems like it's probably only temporary anyway.

Reply:
Yea, I'm pretty sure it's just because I changed the way we publish. These were always 3rd party signed. If we want to clean that up we can, at least this surfaces it somewhere ....